context/AuthContext.tsx

```diff
@@ -1,97 +1,96 @@
 import {createContext, ReactNode, useContext, useEffect, useState} from 'react';
-import {
-    getAuth,
-    onAuthStateChanged,
-    signInWithEmailAndPassword,
-    signOut, updatePassword,
-    User,
-    UserCredential
-} from 'firebase/auth';
+import {getAuth, onAuthStateChanged, signInWithEmailAndPassword, signOut, updatePassword, User} from 'firebase/auth';
 import firebase_app from '@/firebase/config';
-import { getDoc, setDoc} from "@firebase/firestore";
-import { getUserDocument} from "@/util/auth/AuthHelpers";
+import {getDoc, setDoc, updateDoc} from "@firebase/firestore";
+import {getAditionalUserData, getUserDocument} from "@/util/auth/AuthHelpers";
 import {IAditionalUserData} from "@/interfaces/User";
 
 const auth = getAuth(firebase_app);
 
-interface AuthContextType {
-    user: User | null,
-    signInUser: (email: string, password: string) => Promise<void>,
-    signOutUser: () => void,
-    isFirstLogin: boolean | undefined,
-    changePassword: (password: string) => Promise<void>,
-}
-
 const signInUser = async (email: string, password: string) => {
     await signInWithEmailAndPassword(auth, email, password);
 }
 
-
 const signOutUser = async () => {
     return await signOut(auth);
 }
 
-const changePassword = async (password: string) => {
-    return await updatePassword(auth.currentUser as User, password )
+const changePassword = async (password: string, user: User) => {
+    const documentRef = getUserDocument(user);
+    await updatePassword(auth.currentUser as User, password);
+    await updateDoc(documentRef, {isFirstLogin: false})
```

Comment:
My IDE gives an error on this line, but it still seems to run, so not sure if this is a problem

Reply:
what error? I dont get any